packages/webdriver/src/utils.js

```diff
@@ -106,6 +106,16 @@ export function isValidParameter (arg, expectedType) {
     return true
 }
 
+/**
+ * get type of command argument
+ */
+export function getArgumentType (arg) {
+    if (Array.isArray(arg)) {
+        return 'array'
```

Comment:
`array` is not a type

Reply:
Do we want to allow both `[]` or `{}` when expected is supposed to be `object`?